File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Member/ComplianceResult.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

using System.Collections.Generic;

namespace RMA.Service.ClientCare.Contracts.Entities.Member
{
    public class ComplianceResult
    {
        public DebtorStatusEnum? DebtorStatus { get; set; }
        public bool IsBillingCompliant { get; set; }
        public bool IsDeclarationCompliant { get; set; }
        public List<string> Reasons { get; set; }
        public bool IsApplicable { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/compliance-result.ts
import { DebtorStatusEnum } from "projects/fincare/src/app/shared/enum/debtor-status.enum";

export class ComplianceResult {
  debtorStatus: DebtorStatusEnum;
  isBillingCompliant: boolean;
  isDeclarationCompliant: boolean;
  reasons: string[];
  isApplicable: boolean;
}

